resources/istio/files/istio-operator-cluster-evaluation.yaml

```diff
@@ -82,7 +82,10 @@ spec:
           requests:
             cpu: 25m
             memory: 32Mi
+        serviceAnnotations:
+          "service.beta.kubernetes.io/aws-load-balancer-type": "nlb"
         service:
+          externalTrafficPolicy: Local
```

Comment:
This changes the behavior for ALL traffic. And it's not what we may want:

```denotes if this Service desires to route external traffic to node-local or cluster-wide endpoints. “Local” preserves the client source IP and avoids a second hop for LoadBalancer and Nodeport type services, but risks potentially imbalanced traffic spreading. “Cluster” obscures the client source IP and may cause a second hop to another node, but should have good overall load-spreading.```

If you really need that, consider adding another ingress gateway specific for the task. It should be included conditionally, disabled by default for deployments that don't need it.

Reply:
One more thing to add is that we try to keep Istio configuration as close as we can to default Istio configuration. By default this values is set to "Cluster".

Reply:
>  “Cluster” obscures the client source IP and may cause a second hop to another node, but should have good overall load-spreading.`

But for our use case, I think this is exactly what we don't want. We will look at the option with another gateway.